doc/changelog.rst

```diff
@@ -12,6 +12,7 @@ Unreleased
 Core
 ~~~~
 
+- Fix bug: Do not display values in scientific notation
```

Comment:
Can you add word "decimal" to this? I.e. "... display decimal values in ..." To make it easier to find this changelog entry, since I guess "decimal" would be probable keyword people could use to search info about this bug)